execution/executor/src/db_bootstrapper.rs

```diff
@@ -47,13 +47,13 @@ pub fn maybe_bootstrap<V: VMExecutor>(
     db: &DbReaderWriter,
     genesis_txn: &Transaction,
     waypoint: Waypoint,
-) -> Result<bool> {
+) -> Result<Option<LedgerInfoWithSignatures>> {
```

Comment:
Change is no longer relevant. I wanted to say it's okay, but looks like making a clean commit isn't too hard in this case?  Your choice.